src/libraries/Native/AnyOS/System.Globalization.Native/pal_idna.c

```diff
@@ -8,6 +8,13 @@
 #include "pal_icushim_internal.h"
 #include "pal_idna.h"
 
+#ifndef TARGET_UNIX
+// Windows icu headers doesn't define this member.
```

Comment:
>headers [](start = 15, length = 7)

Maybe consider adding a comment that this is omitted as it is marked "deprecated" (as of ICU 55)